data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,417 @@ mission "Remnant: Face to Maw 2B"
+			`	Passing through the cavernous opening into the great spaceport dome, there is a shimmer in the air next to you and a starling fades into view, settling onto the platform beside you. The pilot signs a quick "Greetings, anticipating talking with you." before heading back toward their lock. Behind them, the Starling continues its shutdown checks without any supervision at all.`
+			`	As you finish your own shutdown routine, your commlink informs you that your account has been updated with <payment>.`
+
+
+
+ship "Starling" "Starling (Keystone)"
+		sprite "ship/starling"
+		thumbnail "thumbnail/starling"
+		attributes
+			category "Light Warship"
+			licenses
+				Remnant
+			cost 8800000
+			"shields" 8800
+			"hull" 4900
+			"required crew" 6
+			"bunks" 15
+			"mass" 270
+			"drag" 3.8
+			"heat dissipation" 0.7
+			"fuel capacity" 900
+			"ramscoop" 1
+			"cargo space" 36
+			"outfit space" 312
+			"weapon capacity" 83
+			"engine capacity" 94
+			"shield generation" 1.8
+			"shield energy" 1.2
+			"hull repair rate" 0.8
+			"hull energy" 0.6
+			"cloak" .02
+			"cloaking energy" 5
+			"cloaking fuel" .1
+			"outfit scan power" 12
+			"outfit scan speed" 1
+			"tactical scan power" 26
+			weapon
+				"blast radius" 80
+				"shield damage" 800
+				"hull damage" 400
+				"hit force" 1200
+		outfits
+			"Inhibitor Cannon" 3
+			"Point Defense Turret"
+			
+			"Epoch Cell"
+			"Crystal Capacitor"
+			"Emergency Ramscoop" 2
+			"Quantum Keystone"
+			"Salvage Scanner"
+			
+			"Forge-Class Thruster"
+			"Forge-Class Steering"
+			"Crucible-Class Steering"
+			"Hyperdrive"
+
+		engine -16 82 0.7
+		engine 16 82 0.7
+		engine 0 90 1
+		gun 0 -99 "Inhibitor Cannon"
+		gun -12 -72 "Inhibitor Cannon"
+		gun 12 -72 "Inhibitor Cannon"
+		gun -18 -49
+		gun 18 -49
+		turret 0 2 "Point Defense Turret"
+		leak "remnant leak" 30 4
+		leak "remnant leak sparkle" 30 10
+		explode "tiny explosion" 40
+		explode "small explosion" 20
+		explode "medium explosion" 10
+		"final explode" "final explosion small" 1
+		description "One of the strangest and most useful things that the Remnant have discovered in the abandoned worlds of the Ember Waste is alien cloaking technology. They have designed their scout ships to cloak when needed in order to explore alien territory without being seen... or to escape in a hurry after stealing something particularly valuable from those aliens."
+		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."
```

Comment:
Yes, I should shorten that up a bit.

That being said, no, I have no plans on having any Remnant ships with Hai Keystones be a thing. Quantum Keystones are an interesting thing to the Remnant - but despite what they say, the Keystones only real value is for research purposes. And, as described in the earlier mission dialog, they don't trust them. There's massive security implications in using outfits they don't understand - especially if manufactured deliberately for sale by another faction. 

Aside from testing purposes in this mission, there's no reason to use the Hai Keystone on any other Remnant ship. I suppose to be thorough I could add a follow up mission where we do this exact same experiment with every ship in the Remnant aresenal to see how they each react. (Scientifically, this would be the correct thing to do), but I think it is a case of gameplay vs realism at this point. But beyond this, the Keystone equipped ship won't be used further.

Reply:
Upon making the change and doing some testing; no, I'm not going to shorten it up right now. For some reason, when it is shortened up into the condensed variant format, the NPC loses its "staying" behaviour. I prefer longer-than-strictly-necessary but working over short but not-working.

Reply:
The issue with the ship leaving the system appears to the tied to the variety of personalities used for the NPC, rather than with the definition of the ship (I was able to make it not jump away by replacing the personalities for the ship with the personalities of the Cruiser from FW Reconnaissance). What's noticeable is that the only reason that the ship doesn't leave the system is that it gets locked into a loop of trying to land on one of the planets (through some testing, I was occasionally able to break this behaviour by attacking it, then cloaking, making it jump).